src/performance/perfcollect/perfcollect

```diff
@@ -1217,6 +1229,77 @@ ProcessCollectedData()
 	done
 	WriteStatus "END: Save perf.map files."
 
+	WriteStatus "START: Generate native image map files"
+
+	# Get the list of loaded images and use the path to libcoreclr.so to find crossgen.
+	# crossgen is expected to sit next to libcoreclr.so.
+	local buildidList=`$perfcmd buildid-list | grep libcoreclr.so | cut -d ' ' -f 2`
+	local crossgenCmd=''
+	local crossgenDir=''
+	for file in $buildidList
+	do
+		crossgenDir=`dirname "${file}"`
+		if [ -f ${crossgenDir}/crossgen ]
+		then
+			crossgenCmd=${crossgenDir}/crossgen
+			echo "Found crossgen at ${crossgenCmd}"
+			break
+		fi
+	done
+
+        OLDIFS=$IFS
```

Comment:
Can you fix the intending here?  Looks like you have an extra space on a few lines.